src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -163,7 +163,8 @@ def initialize_jobs(self, cell_ids: List[str] = None) -> None:
 
             # Set to refresh when job is not in terminal state
             # and when job is present in cells (if given)
-            refresh = not job.was_terminal()
+            # and when it is not part of a batch
+            refresh = not job.was_terminal() and not job.batch_id
```

Comment:
all jobs in a batch have a batch ID, but it is set to `None` in non-batch jobs

Reply:
Easiest change ever!